containers/podman-desktop
# chore: ensure extensions have valid typescript

### What does this PR do?

I noticed that if extensions have invalid code, JS code is still emitted while we should abort the build


FYI here is an example where it should have failed, these extensions using new API that is not available so PR should be red
https://github.com/containers/podman-desktop/pull/4049

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

exemple of code
https://github.com/containers/podman-desktop/pull/4049

### How to test this PR?

Use an invalid extension API in the extension and check `yarn typecheck` fails the build
